src/deepsparse/v2/text_generation/nl_engine_operator.py

```diff
@@ -29,7 +31,12 @@
 )
 
 
-__all__ = ["NLEngineOperator", "NlEngineInput"]
+__all__ = [
+    "NlEngineOperator",
+    "NlEngineOperatorNoCache",
```

Comment:
this should be its own operator in a separate file

Reply:
Why can't I create variations of the same operator in the same script? I mean it makes sense for them to "live close" to each other. This question is about both NlEngineOperator(NoCache) as well as TextGenerationPipeline(NoCache)

Reply:
Come on

Reply:
⛔

Reply:
⚔️